web/src/compose_closed_ui.js

```diff
@@ -15,7 +15,7 @@ export function get_recipient_label(message) {
     // passed in.
 
     if (message === undefined) {
-        if (message_lists.current.visibly_empty()) {
+        if (message_lists.current?.visibly_empty()) {
```

Comment:
I think in this function, just doing at the top of the function an `if (message_lists.current === undefined) { return "" };` is probably easier to reason about? 

(It's not at all obvious why one should treat it as "not visibly empty", for example)

Reply:
Okay, made that change.